tests/AutoReview/FixerTest.php

```diff
@@ -235,11 +239,12 @@ private static function assertStringIsNotEmpty($actual, $message = '')
     }
 
     /**
-     * @param string $actual
+     * @param string $haystack
+     * @param string $needle
      * @param string $message
      */
-    private static function assertPhpUnitCasing($actual, $message)
+    private static function assertCorrectCasing($haystack, $needle, $message)
```

Comment:
expected value, here expected casing, shall be first param, to mimic native PHPUnit's assertions params order